sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/slack.yaml

```
@@ -102,4 +102,51 @@ provider:
           containsSecret: false
       detection:
         matching:
-          pattern: "((https://)?hooks\\.slack\\.com/workflows/T[A-Za-z0-9+\\/]{42,70})"
\ No newline at end of file
+          pattern: "((https://)?hooks\\.slack\\.com/workflows/T[A-Za-z0-9+\\/]{42,70})"
+
+    - rspecKey: S6731
+      id: slack-bot-tokens
+      metadata:
+        name: Slack Bot Token
+        message: Make sure this Slack Bot Token gets revoked, changed, and removed from the code.
+      detection:
+        pre:
+          include:
+            content:
+              # Slack Bot Token
+              - xoxb
+              # Slack User Token
+              - xoxp
+              # Slack Workspace Access Token
+              - xoxa
+              # Slack Workspace Refresh Token
+              - xoxr
+
+        matching:
+          # Type 1 secret, fix xox[bpar]-* pattern
+          pattern: "(xox[bpar]\\-[0-9]{10,13}\\-[0-9]{10,13}-\\w+)"
```

```suggestion
          pattern: "\\b(xox[bpar]\\-[0-9]{10,13}\\-[0-9]{10,13}(?:\\-[0-9]{10,13})?-\\w+)\\b"
```
It looks like there are some secrets with three dashes.